zipkin-collector-service/build.gradle

```diff
@@ -1,6 +1,13 @@
 apply plugin: 'application'
 mainClassName = 'com.twitter.zipkin.collector.Main'
 
+run {
+    // When configKey is dev, collector and query share ../zipkin.db
+    workingDir rootProject.projectDir
```

Comment:
best way out was to point query to collector's build dir when configKey == "dev". The fundamental relationship is the coupling of these two processes, not to the root project. Ends up less work, too.